src/main/kotlin/build/buf/intellij/annotator/BufAnalyzeUtils.kt

```diff
@@ -185,12 +191,25 @@ object BufAnalyzeUtils {
         )
         handler.addProcessListener(object : ProcessAdapter() {
             override fun onTextAvailable(event: ProcessEvent, outputType: com.intellij.openapi.util.Key<*>) {
-                result.add(event.text.trimEnd())
+                when (outputType) {
+                    ProcessOutputType.STDOUT -> stdout.add(event.text.trimEnd())
+                }
+            }
+
+            override fun processTerminated(event: ProcessEvent) {
+                exitCode.set(event.exitCode)
```

Comment:
We were never checking the exit code of the process that I could tell. This will make sure we add better logging around this.